cl_dll/c_playerresource.h

```diff
@@ -1,4 +1,4 @@
-//========= Copyright � 1996-2005, Valve Corporation, All rights reserved. ============//

```

Comment:
?

Reply:
I will restore this and other instances I see